### PR TITLE
CONN-2914 Add New JFrog Cloud-To-Cloud Source

### PR DESCRIPTION
**Pull request by carlos-castillo-jask**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

CONN-2914

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/jfrog-xray/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/jfrog-xray/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/jfrog-xray/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/security-threat-detection/jfrog.png')} alt="thumbnail icon" width="85"/>
+
+The JFrog Xray source will collect JFrog Xray violations by querying the API for new policy violations when they occur.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:19]:
```suggestion
The Sumo Logic JFrog Xray source collects JFrog Xray violations by querying the API for new policy violations when they occur.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+| Polling Interval | Data |
+| :--- | :--- |
+| 5 minutes |  [Get Violations Endpoint](https://jfrog.com/help/r/xray-rest-apis/get-violations)|
+
+
+## Setup
+
+### Vendor configuration
+
+Ensure Xray has policies configured. This source will ingest violation logs when those policies raise violations. You will need the domain of your JFrog instance and JFrog username/password in order to configure this source. We suggest using a dedicated user account for this source.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:36]:
```suggestion
Make sure that you've set up Xray's policies correctly. This source will capture violation logs triggered by those policies. To configure this source, you'll need the domain of your JFrog instance and your JFrog username/password. We recommend creating a dedicated user account specifically for this purpose.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Source configuration
+
+When you create a JFrog Xray Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure the JFrog Xray Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+1. On the Collectors page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **JFrog Xray** icon.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:45]:
```suggestion
1. Search for and select the **JFrog Xray** icon.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+1. Enter a **Name** to display for the Source in the Sumo Logic web application. The description is optional.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:46]:
```suggestion
1. Enter a **Name** to display for the Source in Sumo Logic. The description is optional.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`. 
+1. (Optional) **Fields**. Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. In **JFrog Base URL**, enter your Jfrog instance domain. E.g. `https://acme.jfrog.io`
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:51]:
```suggestion
1. In **JFrog Base URL**, enter your JFrog instance domain (for example, `https://acme.jfrog.io`).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+1. In **HTTP Basic Auth Username**, enter your JFrog username you created
+1. In **HTTP Basic Auth Password**, enter your JFrog password you created
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:53]:
```suggestion
1. In **HTTP Basic Auth Username**, enter your JFrog username you created.
1. In **HTTP Basic Auth Password**, enter your JFrog password you created.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+1. In **Collect Violation Details**, only toggle this on if you need the full violation details. This will require 1 extra API call per violation log.
+1. (Optional) The **Polling Interval** is set for 5 minutes default, you can adjust it based on your needs.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray.md:55]:
```suggestion
1. (Optional) The **Polling Interval** is set for 5 minutes by default. You can adjust it based on your needs.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting stuff

**Comment by carlos-castillo-jask** [comment]
Thanks for the suggestions Kim. I think I need to also add this to the sidebar link.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 8]
```diff
@@ -276,6 +276,12 @@ In this section, we'll introduce the following concepts:
   <p>Collects a list of users from the Google Workspace Users API.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jfrog-xray-source"><img src={useBaseUrl('img/integrations/security-threat-detection/jfrog.png')} alt="icon" width="60"/><h4>JFrog Xray</h4></a>
+  <p>Collect violation events from JFrog Xray</p>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:282]:
```suggestion
  <p>Collect violation events from JFrog Xray.</p>
```

**Review by kimsauce — APPROVED** [review]
Approved contingent on minor fix

**Pull request merged**